application/src/main/java/org/opentripplanner/apis/transmodel/TransmodelGraphQLSchema.java

```diff
@@ -1300,40 +1302,19 @@ private GraphQLSchema create() {
               .build()
           )
           .dataFetcher(environment -> {
-            List<FeedScopedId> lineIds = mapIDsToDomainNullSafe(
-              environment.getArgumentOrDefault("lines", List.of())
-            );
-            List<String> privateCodes = environment.getArgumentOrDefault("privateCodes", List.of());
-            List<LocalDate> activeServiceDates = environment.getArgumentOrDefault(
-              "activeDates",
-              List.of()
-            );
+            var authorities = mapIDsToDomainNullSafe(environment.getArgument("authorities"));
```

Comment:
This changes the semantics when filtering by authorities since it was previously filtered on the pure id instead of the feed scoped id. The new behaviour is more correct but it is not backwards compatible.

I'll have to check if we are using this at skånetrafiken. Are you ok at entur with changing this behaviour?

Reply:
I double-checked that this works at Entur. We have a fixedFeedId that gets prefixed to the strings passed through this method[1]. Since the test only compares the id field on the feedScopedId this works fine for us. I suppose this might not work if someone does not use a fixedFeedId, but I don't know if that is the case or if it is even possible - do you know more about this?

I'll take a closer look at how the fixedFeedId is set too.

[1]https://github.com/opentripplanner/OpenTripPlanner/blob/46be411361388514c21453e0d6162528d4df8834/application/src/main/java/org/opentripplanner/apis/transmodel/mapping/TransitIdMapper.java#L52-L54
<!-- Reviewable comment -OB_e6T4A8wNDd5EQ0Uq:b2vuhxe -->
<!-- Sent from Reviewable.io -->

Reply:
We don't use a fixedFeedId at skånetrafiken, I didn't even know about that feature. So I had to look into this, but it seems like in practice we are currently not using the authority filtering on this endpoint. So we are fine with this breaking change.

Reply:
We need to test this, I have a vage memory that we have not corrected this because it would break - I think I accidentally fix it at some point and had to revert it.  But, not sure. Test and fix it if we can.

Reply:
Looks like it's working fine in Entur dev: https://api.dev.entur.io/graphql-explorer/?query=%7BserviceJourneys%28authorities%3A%22TEL%3AAuthority%3ATFK_ID%22%29%7Bline%7Bauthority%7Bid%7D%7D%7D%7D